samples/csharp_dotnetcore/14.nlp-with-dispatch/README.md

```diff
@@ -89,13 +88,13 @@ To create the LUIS application this bot needs and update the .bot file configura
 - Navigate to botbuilder-samples/csharp_dotnetcore/samples/14.nlp-with-dispatch
 - Run the following commands
 ```bash 
-> ludown parse toluis --in Resources/homeautomation.lu -o CognitiveModels --out homeAutomation.luis -n "Home Automation" -d "Home Automation LUIS application - Bot Builder Samples" --verbose
+> ludown parse toluis --in Resources/homeautomation.lu -o CognitiveModels --out homeautomation.luis -n "Home Automation" -d "Home Automation LUIS application - Bot Builder Samples" --verbose
 
 > ludown parse toluis --in Resources/weather.lu -o CognitiveModels --out weather.luis -n Weather -d "Weather LUIS application - Bot Builder Samples" --verbose
 
-> luis import application --in CognitiveModels/homeAutomation.luis --authoringKey <LUIS-AUTHORING-KEY> --region <LUIS-AUTHORING-REGION> --msbot | msbot connect luis --stdin
+> luis import application --in CognitiveModels/homeautomation.luis --authoringKey <LUIS-AUTHORING-KEY> --subscriptionKey <LUIS-SUBSCRIPTION-KEY> --region <LUIS-AUTHORING-REGION> --msbot | msbot connect luis --stdin
```

Comment:
I'm not sure why, but this step produced a "Home Automation" app on the LUIS.ai site, instead of a "homeautomation.luis" app.